app/Listeners/StateHistoryManager.php

```diff
@@ -33,7 +33,7 @@ public function postTransition(TransitionEvent $event)
 
         $model->history()->create([
             "transition" => $event->getTransition(),
-            "from" => "", // what the state was before.
+            "from" => $event->getState(), // what the state was before.
```

Comment:
Will it be empty if there wasn't one? I think there was a trello card about this way back when - to implement when we started to care that the `from`s were not being recorded. I guess you can locate and annihilate?

Reply:
yes, I think it pops out null.